runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -105,9 +105,12 @@
 
 	// Chest loot handling
 	private static final String CHEST_LOOTED_MESSAGE = "You find some treasure in the chest!";
+	private static final Pattern LARRAN_LOOTED_PATTERN = Pattern.compile("You have opened Larran's ([a-z]+) chest .*");
```

Comment:
just use big|small instead of this, or?

Reply:
Made the change. My thinking was if they add a different chest, like a medium, it wouldn't need to be updated again.